challenge_2/cpp/dewie102/README.md

```diff
@@ -0,0 +1,21 @@
+# Reverse a String
```

Comment:
Yes I copied over the file to get the layout and forgot to change it, thank you for pointing that out.